### PR TITLE
Implemented error component

### PR DESCRIPTION
**Pull request by richardzeng1**
Red block appears at top of page with error message.
All components must implement `this.$emit('error', message)` where message is the error message.
This method sends the error message back to the parent component.

### CODE REVIEW

**Comment by richardzeng1** [comment]
fixes #50

File: src/views/components/errorBlock.vue [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<div id="error-block">
+    <p>Error: {{error_message}}</p>
+</div>
+
+</template>
+
+<script>
+export default {
+    name: "edit-courses",
+    props:["error_message"]
```
Comment (siefkenj) [inline src/views/components/errorBlock.vue]: pretteir

File: src/App.vue [hunk 2]
```diff
@@ -9,19 +9,37 @@
...
+        sendError:function(value){
+            this.error_message=value,
+            this.error = true;
+            console.log(value);
+        }
+    },
+    data:function(){
+        return{
+            error:false,
+            error_message:""
```
Comment (siefkenj) [inline src/App.vue]: prettier

File: src/App.vue [hunk 3]
```diff
@@ -9,19 +9,37 @@
...
 
 <script>
-import Sidenav from './views/components/navbar.vue'
+import Sidenav from './views/components/navbar.vue';
+import ErrorBlock from './views/components/errorBlock.vue';
 export default {
     name: "app",
+    methods:{
+        sendError:function(value){
+            this.error_message=value,
```
Comment (siefkenj) [inline src/App.vue]: don't use the comma operator.

**Pull request merged**